src/App.tsx

```diff
@@ -1,24 +1,255 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import {
+  createTodo, getTodos, deleteTodo, updateTodo,
+} from './api/todos';
+import { Footer } from './components/Footer/Footer';
+import { Header } from './components/Header/Header';
+import { Main } from './components/Main/Main';
+import { ErrorMessages } from './components/ErrorMessages/ErrorMessages';
+import { ErrorTypes } from './types/ErrorTypes';
+import { Status } from './types/Status';
 
-const USER_ID = 0;
+const USER_ID = 10548;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<Status>(Status.default);
+  const [disableInput, setDisableInput] = useState(false);
+  const [errorMessage, setErrorMessage] = useState<ErrorTypes | null>(null);
+  const [hasEditTodo, setHasEditTodo] = useState(false);
+  const [todoForUpdate, setTodoForUpdate] = useState<Todo | null>(null);
+  const [idTodoForCheange, setIdTodoForSpinner] = useState<number[]>([]);
```

Comment:
```suggestion
  const [idTodoForChange, setIdTodoForChange] = useState<number[]>([]);
```
fix type and follow of general convention in naming setState function in useState